src/EnergyPlus/CurveManager.hh

```diff
@@ -107,6 +106,29 @@ namespace CurveManager {
         Num
     };
 
+    constexpr std::array<std::string_view, static_cast<int>(CurveType::Num)> objectNames = {
+        "Curve:Linear",
+        "Curve:Quadratic",
+        "Curve:Biquadratic",
+        "Curve:Cubic",
+        "Curve:QuadLinear",
+        "Curve:Bicubic",
+        "Curve:Triquadratic",
+        "Curve:Exponent",
+        "Curve:Quartic",
+        "Curve:FanPressureRise",
+        "Curve:ExponentialSkewNormal",
+        "Curve:Sigmoid",
+        "Curve:RectangularHyperbola1",
+        "Curve:RectangularHyperbola2",
+        "Curve:ExponentialDecay",
+        "Curve:DoubleExponentialDecay",
+        "Curve:QuadraticLinear",
+        "Curve:QuintLinear",
+        "Curve:CubicLinear",
+        "Curve:ChillerPartLoadWithLift",
```

Comment:
Table:Lookup is not a curve type?

Reply:
I think it will be after we add some inheritance to this bad boy.